### PR TITLE
fix: some typo

### PR DESCRIPTION
**Pull request by sailingwithoutwind**

### CODE REVIEW

File: contributors/design-proposals/apps/controller_history.md [hunk 1]
```diff
@@ -398,7 +398,7 @@ as an extension of the [above](#viewing-history ).
 ```
 
 Here `kubectl undo` simply uses strategic merge patch to apply the state 
-contained at a particular revision.
```
Comment (cblecker) [inline contributors/design-proposals/apps/controller_history.md]: I think "at" is intended here.

**Comment by sailingwithoutwind** [comment]
1

File: contributors/design-proposals/apps/controller_history.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ and it can be solved by applying a version control system to manifests. Internal
 revision history must only store revisions to the controller's target state that 
 correspond to live Objects and (potentially) a small, configurable number of 
 prior revisions.
-1. History is scale invariant. A revision to a controller is a modification 
+1. History is scaled invariant. A revision to a controller is a modification
```
Comment (prydonius) [inline contributors/design-proposals/apps/controller_history.md]: I believe "scale invariant" is correct, but correct me if I'm wrong.

Comment (cblecker) [reply contributors/design-proposals/apps/controller_history.md]: Yes, I believe what was there is correct too

**Comment by sailingwithoutwind** [comment]
sign CLA

**Pull request merged**